### DISCUSSION TITLE
Save Trained Model

### DISCUSSION THREAD

**Discussion by RotoDrache** [Project-MONAI/MONAILabel#1238] [Q&A]
Hi I apologize as my question may be very basic to most of you. I have performed a training from scratch as per the PROJECT Monai video on a custom dataset(app radiology model segmentation). 

Can I now save this model somehow and call upon it at a later date to autosegment a . If so can you direct me to the instructions.

Many thanks

**Comment by tangy5** [comment]
If the training completes, and validation data are correctly set. The latest trained model can be automatically saved to  "radiology/models" folder. You can find the model checkpoint there and use/call it at later time. 

Thanks.

**Comment by RotoDrache** [comment]
OK super thank-you once again. Ill continue train and check for the model
